app/components/Forms/ConnectionForm.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useRef } from 'react';
+import { useAppSelector, useAppDispatch } from '@store/store';
+import Button from '@components/Input/Button';
+import { TextInputRef, Toggle } from '@components/Input';
+import { toggleConnectionModal } from '@store/slices/global';
+import { toggleIsPWVisible, connection } from '@store/slices/connection';
+import { mutationRequestCSR } from '@utils/fetchApi';
+import { login } from '@store/slices/user';
+import { toast } from 'react-hot-toast';
+import tryCatchWrapper from '@utils/tryCatchWrapper';
+import type { BodyData } from '@utils/fetchApi';
+
+export default function ConnectionForm() {
+  const isPWVisible = useAppSelector(connection).isPWVisible;
+  const dispatch = useAppDispatch();
+
+  const identifierRef = useRef<HTMLInputElement>(null);
+  const passwordRef = useRef<HTMLInputElement>(null);
+  const allInputsRef = [identifierRef, passwordRef];
+
+  const submitSuccess = async (method: 'POST' | 'PUT' | 'DELETE', endpoint: string, data?: BodyData) => {
+    const responseData = await mutationRequestCSR(method, endpoint, data);
+    // Send a confirmation toast -> To do later
+    toast.success(responseData.response);
+    // State Mutation
+    dispatch(login(responseData.user));
+    // Save the accessToken in the localStorage
+    window.localStorage.setItem('accessToken', responseData.token);
+    // Closing modal
+    dispatch(toggleConnectionModal());
+  };
+
+  const handleSubmit = async(e: React.FormEvent<HTMLFormElement>, allInputsRef: React.RefObject<HTMLInputElement>[]) => {
+    e.preventDefault();
+    // Passing all inputs as required
+    allInputsRef.forEach((inputRef) => {
+      if (inputRef.current) inputRef.current.required = true;
+    });
+    // Checking all inputs validation status
+    const inputValidationStatus = allInputsRef.map((inputRef) => inputRef.current?.reportValidity());
+    if (!inputValidationStatus.includes(false)) {
+      const data = {
+        password: passwordRef.current!.value,
+        pseudo: identifierRef.current!.value,
+      };
+      tryCatchWrapper(submitSuccess)('POST', '/login', data);
+    }
+  };
```

Comment:
If you name one *handler* with *handle* prefix you should do that with all handlers.
`submitSuccess` => `handleSubmitSuccess`

Reply:
Issue #71